### ISSUE TITLE
Some requests about the UI

### ISSUE THREAD

**Issue by ApostolosB** [vtorri/etui#15]
Since it is being rewritten i think i should ask for some features.

A mini view of the pages on the left or right of the window.
Tabs so you can have many docs open.
Side by side view of many tabs (ie hold a key and click on the tabs to show them on the same win)

**Comment by vtorri** [comment]
imho, the miniview should only be shown on demand. I want to use the maximum part of the windows to view the doc
And I also want to have an exposé view
I also wanted to use tabs
for your last request, hmm, for what purpose ?

**Comment by ApostolosB** [comment]
Miniview obviously on demand only. Like the terminology miniview. 

The last request can come handy when you have multiple docs and you want to compare stuff etc. Ie specs of products or measurements or whatever. Going back and forth by clicking through tabs is a bit cumbersome.

**Comment by ApostolosB** [comment]
One more thing. Is it possible to have all the supported docs of a directory (chosen by the user) showing up when you start etui from the menu?

**Comment by vtorri** [comment]
i prefer showing directly a file selector if no file is provided, maybe also a possibility to reload all the files opened in the previous session of etui

**Comment by ApostolosB** [comment]
Files from the previous session might have been deleted or moved so it could be problematic. My idea came from what we have in rage (the video player) where when you open it without a file it shows thumbnails of all the videos you have in /Videos.

Anyway. Your code :)

**Comment by ApostolosB** [comment]
@vtorri saw you added the file selector. Is it supposed to be fully functional? Cause it doesn't seem to work for me.

**Comment by vtorri** [comment]
no, not yet, just the gui for now. As I plan to have several docs, i am writing the code to handle that. When it's done, I link the fileselector with the docs
